### ISSUE TITLE
404 error when trying to use Lifecycle functions

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#38]
It's inevitable that someone will run into this, so figured I'd document it here. If you try to import something from the base svelte package, like the documentation tells you to do for a lifecycle function like [onMount](https://svelte.dev/tutorial/onmount):

```html
<script>
  import { onMount } from 'svelte';
</script>
```
You will get a 404 error. This is due to Gopack not being smart enough to read this reference. You can get around this by simply referencing the exact package, which is "internal" in this case:

```html
<script>
  import { onMount } from 'svelte/internal';
</script>
```

<details>
<summary>Here are the errors you will see in the browser</summary>
<br>

Error in Firefox:
```
Loading module from “http://localhost:3000/spa/web_modules/svelte/internal/” was blocked because of a disallowed MIME type (“text/html”).
```
Error in Chrome:
```
Failed to load module script: The server responded with a non-JavaScript MIME type of "text/html". Strict MIME type checking is enforced for module scripts per HTML spec.
```

</details>

**Comment by jimafisk** [comment]
The problem here is we're trying to walk nested directories to find the file. The "svelte" base folder does have an `index.js` file that exports everything from the `internal/` folder, but we start walking the nested directories before we get to that file, and find the `index.js` files within the nested folder first. 

We do need to still check the subdirectories for libs that use a `dist/` folder, like navaid, but we only want to do that after check the current directory for a file with `.js` extension.

We can check through current dir with `ioutil.ReadDir()`: https://stackoverflow.com/questions/14668850/list-directory-in-go

**Comment by jimafisk** [comment]
Fixed in [v0.3.15](https://github.com/plentico/plenti/releases/tag/v0.3.15)
